src/main/resources/messages_template.xml

```diff
@@ -689,6 +689,17 @@
     <reason>Conref range: Unable to find conref range end element with ID "%1".</reason>
     <response/>
   </message>
+
+  <message id="DOTX072W" type="WARN">
+    <reason>Failed to resolve topic ID "%1" to a topic in document "%2".</reason>
```

Comment:
The location of error condition should not be part of the error message, it's populated from `@xtrf`.